src/main/java/edu/cornell/kfs/concur/rest/xmlObjects/ExpenseDetailedReportDTO.java

```diff
@@ -0,0 +1,5 @@
+package edu.cornell.kfs.concur.rest.xmlObjects;
+
+public class ExpenseDetailedReportDTO {
```

Comment:
Is the plan to implement this class later? It seems odd to include an empty class here.